mod/wikirate_source/set/type/source/source_item.rb

```diff
@@ -156,12 +155,13 @@ def hidden_item_input
   end
 
   view :source_and_preview, cache: :never do |args|
-    wrap_with :div, class: "source-details",
-                    data: { source_for: card.name, year: year } do
-      args[:url] = source_url
-      with_cite_button +
-        render_iframe_view(args).html_safe +
+    args[:url] = source_url
```

Comment:
fwiw, doesn't look to me like either of those views does anything with args[:url]